client/src/components/Contact.js

```diff
@@ -1,14 +1,40 @@
-import React, { Component, Fragment } from 'react';
+import React, { Fragment } from 'react';
+import ReactRouterPropTypes from 'react-router-prop-types';
+import * as S from './Contact.style';
 import CHeader from './CHeader';
+import CFooter from './CFooter';
 
-class Contact extends Component {
-  render() {
-    return (
-      <Fragment>
-        <CHeader menu />
-      </Fragment>
-    );
-  }
-}
+const Contact = () => {
+  return (
+    <Fragment>
+      <CHeader menu />
+      <S.Main>
+        <S.H1>Contact</S.H1>
+        <S.ContactWrapper>
+          <S.H3>
```

Comment:
this should actually be a h2!
headings should always read in order, so you shouldn't have a h1 and h3 without a h2 :)
This is helpful for screen readers.

Also, you can remove the ```<u>``` and use a 
```text-decoration: underline``` in the styled component!
https://www.w3schools.com/cssref/pr_text_text-decoration.asp